### PR TITLE
chore(#8952): add l800-l1000 Swahili translations

### LINKED ISSUE #8952: Add Swahili Translations L800 to L1000
Related to https://github.com/medic/cht-core/issues/8884

#### What feature do you want to improve?
Some translation keys are missing Swahili entries.

#### Describe the improvement you'd like
Add Swahili entries to translation keys.

### PR DESCRIPTION
**Pull request by Benmuiruri**
# Description

Add L800 - L1000 Swahili translations

Closes #8952 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8952-add-l800-l1000-swahili-translations/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8952-add-l800-l1000-swahili-translations/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8952-add-l800-l1000-swahili-translations/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by Benmuiruri** [comment]
Hi @esthermmoturi please counter-check my Swahili accuracy when you have some time. Thanks 😄

File: api/resources/translations/messages-sw.properties
Comment (esthermmoturi) [inline api/resources/translations/messages-sw.properties:980]:
```suggestion
permission.description.can_bulk_delete_reports = Inaruhusiwa kutumia vipengele vya kufuta kwa wingi ili kufuta ripoti nyingi
```

**Review by esthermmoturi — APPROVED** [review]

**Comment by Benmuiruri** [comment]
Hi @njuguna-n please approve this Pre-approved PR by Mariachana 😃 . I need approval from reviewer with write access. Thanks

**Review by njuguna-n — APPROVED** [review]

**Pull request merged**
